diff --git a/dist/manager-helpers.js b/dist/manager-helpers.js
index ff31444cee1331fd65b2d4e74da7355ed2343500..5ee515e4478fafa04ee9cf4fdd09115f4386b1db 100644
--- a/dist/manager-helpers.js
+++ b/dist/manager-helpers.js
@@ -13,7 +13,7 @@ import { NAVIGATE_URL, GLOBALS_UPDATED, STORY_ARGS_UPDATED, UPDATE_STORY_ARGS, R
 import { filterArgTypes } from 'storybook/preview-api';
 import { SourceType } from 'storybook/internal/docs-tools';
 import { addons, Tag } from 'storybook/internal/preview-api';
-import 'storybook/internal/channels';
+// import 'storybook/internal/channels';
 
 var CustomBadge = (props) => {
   return /* @__PURE__ */ React9__default.createElement(Badge, { context: "mdx", ...props });
